src/client/features/view/components/sidebar/menus/components/downloadOption.js

```diff
@@ -23,7 +24,11 @@ class DownloadOption extends React.Component {
     };
   }
 
-  handleDownloadClick(type) {
+  handleDownloadClick(evt, type) {
+    // Don't do anything when a link is clicked. Should not download just because
+    // user clicks a link
+    if (evt.target.tagName === 'A') return;
```

Comment:
Try:
```js
evt.preventDefault();
```
Instead of checking for the tagName and see if it works.

Reply:
`evt.preventDefault()` doesn't work in this case it would seem because it tries to prevent the default behaviour of the event. In this case I want the default behaviour and not the behaviour I define, so if a user clicks a link inside the download option, the link opens, but the download does not happen.

Reply:
So when a user clicks an 'a' tag, you want to open the link and download a file?

Reply:
No, I want to open the link and not download the file.

Reply:
Do you always want to open the link and not download the file or does that depend on the type of file you are downloading?

Reply:
Doesn't matter the file type. The links are clickable without downloading the file. The whole card is clickable for the download except the links.